pallets/frequency-tx-payment/src/lib.rs

```diff
@@ -441,17 +461,36 @@ where
 	) -> Result<(BalanceOf<T>, InitialPayment<T>), TransactionValidityError> {
 		let mut calls_weight_sum = Weight::zero();
 		for call in calls {
-			let call_weight = T::CapacityCalls::get_stable_weight(call)
-				.ok_or(ChargeFrqTransactionPaymentError::CallIsNotCapacityEligible.into())?;
+			let call_weight = if self.call_is_adding_eligible_key_to_msa(call) {
+				// "free-ish"
+				Weight::from_parts(1_000, 0)
```

Comment:
This doesn't seem to work out to much of a discount. See e2e test.

Reply:
Per conversation in Wednesday's meeting, rather than worrying with the weight calculation at all, we will just halve the calculated capacity fee.